web/modules/custom/usagov_ssg_postprocessing/src/EventSubscriber/TomeEventSubscriber.php

```diff
@@ -132,17 +133,22 @@ public static function getExcludedDirectories() {
    */
   public function modifyHtml(ModifyHtmlEvent $event) {
     $html = $event->getHtml();
+    $html5 = new HTML5();
 
     // LIBXML_SCHEMA_CREATE fixes a problem wherein DOMDocument would remove closing HTML
     // tags within quoted text in a script element. See https://bugs.php.net/bug.php?id=74628
     $document = new \DOMDocument();
     @$document->loadHTML($html, LIBXML_SCHEMA_CREATE);
```

Comment:
Did you try using `$html5->loadHTML($html)`? 
It looks like you can replace these two lines with `$document = $html5->loadHTML($html);` 
(I'm looking at https://github.com/Masterminds/html5-php/wiki/Basic-Usage)

It seemed to me that using the HTML5 parser should be better than using php's built-in libxml, since it actually supports HTML5.

Reply:
Okay, thanks for checking! I'm surprised that just making this change on the output end solves our problem, but it definitely does.